### ISSUE TITLE
Problem: Please help... Response has started

### ISSUE THREAD

**Issue by Andale-online** [elsa-workflows/elsa-core#152]
I get an Fault "Response has already started" while sitting at the XMLHandler...

Tried with ExecuteAsync and Execute .... Same result

```
            var response = httpContextAccessor.HttpContext.Response;

            if (response.HasStarted)
                return Fault("Response has already started");
```

What the...?

**Comment by sfmskywalker** [comment]
Do you have a link for me to the source code? I tried to find it in your fork, but I gave up 🤷‍♂

**Comment by Andale-online** [comment]
Damned, had to reorganize /rebase my fork and local git - and exactly in these 20mins you accessed... 8-)

**Comment by sfmskywalker** [comment]
Well, at least that gave me a giggle 😄

**Comment by Andale-online** [comment]
:-D

I marked the position in the PR

**Comment by Andale-online** [comment]
This can be closed and is followed up by: #160
